native-emulation/install.sh

```diff
@@ -51,7 +51,7 @@ extract_images () {
     curl -sSL "$IMAGE_URL" -o "${IMAGE}.zip"
   [ -f "${IMAGE}.img" ] || \
     unzip "${IMAGE}.zip"
-  [ -f "${IMAGE}.img" ] || \
+  [ -f "${IMAGE}.img" ] && \
```

Comment:
We want to resize the image file if the file exists, not if it doesn't exist.

Reply:
thanks!